services/QuillLMS/app/services/analytics/segment_analytics.rb

```diff
@@ -200,6 +200,20 @@ def track_school_subscription(subscription, event)
     end
   end
 
+  def track_teacher_school_not_listed(user, school_name, zipcode)
+    track({
+      # Segment requires us to send a unique User ID or Anonymous ID for every event
+      # generate a random UUID here because we don't want the School Subscription event to be associated to any real user
```

Comment:
Looks like this comment is leftover from a copy/paste, but I don't think it's accurate here.